src/style.css

```diff
@@ -0,0 +1,139 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable declaration-block-trailing-semicolon */
```

Comment:
Don't disable the eslint. Fix the problems with it instead. Try googling the error text